.github/dependabot.yml

```diff
@@ -0,0 +1,53 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "11:00"
+    timezone: "America/Los_Angeles" # Pacific Time
+  labels:
+  - "category: engineering"
+  - dependencies
+  versioning-strategy: widen
+  ignore:
+    # axe-core updates require specific engineering work to accept. See
+    # README.md for instructions.
+  - dependency-name: axe-core
+    versions:
+    - ">= 0"
+    # Major version of @types/node is pinned to match the version of node we
+    # use for builds (ideally, latest LTS)
+  - dependency-name: "@types/node"
+    versions:
+    - ">=15.0.0"
```

Comment:
I'm unsure whether this should be 14 because we use 14 in the builds. Is 15 still compatible?